conan/tools/meson/__init__.py

```diff
@@ -1,2 +1,3 @@
 # noinspection PyUnresolvedReferences
-from .meson import MesonToolchain
+from .toolchain import MesonToolchain
```

Comment:
Start using absolute imports everywhere. It was wrong in other tools, but it is been fixed already.

Reply:
changed to use absolute imports